### PR TITLE
Add one liner for uploading and downloading from our private PyPI

### PR DESCRIPTION
**Pull request by akaszynski**
Pinging @kellymore for her review.

Badly needed based on user interaction.

### CODE REVIEW

File: doc/source/how-to/releasing.rst [hunk 1]
```diff
@@ -368,6 +377,14 @@ get support during the first release of the project. The team then enables the
 custom ``PYPI_TOKEN`` once your project has been successfully released for the
 first time. For future releases, everything is then automated.
 
+Here's a one liner for downloading:
+
+.. code::
+
+   python -m pip install <PACKAGE-NAME> --index-url <TOKEN-REDACTED>@pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/simple/
+
+Replace ``<PACKAGE-NAME>`` and ``<TOKEN-REDACTED>`` with the package name and the private PyPI token repectively.
```
Comment (akaszynski) [inline doc/source/how-to/releasing.rst:386]:
```suggestion
Replace ``<PACKAGE-NAME>`` and ``<TOKEN-REDACTED>`` with the package name and the private PyPI token respectively.
```

File: doc/source/how-to/releasing.rst [hunk 2]
```diff
@@ -271,6 +271,15 @@ Forked GitHub repositories do not have access to GitHub secrets. This is
 designed to protect against pull-requests that could potentially scrape
 tokens from PyAnsys CI/CD.
 
+Here's a cross-platform one liner for uploading using ``twine```:
+
+.. code::
+
+   python -m twine upload dist/* --repository-url https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload -u __token__ -p <TOKEN-REDACTED>
+
+Replace ``<TOKEN-REDACTED>`` with the private PyPI token repectively.
```
Comment (akaszynski) [inline doc/source/how-to/releasing.rst:280]:
```suggestion
Replace ``<TOKEN-REDACTED>`` with the private PyPI token respectively.
```

**Review by MaxJPRey — APPROVED** [review]

**Review by kellymore — COMMENTED** [review]
because `set TWINE_PASSWORD=<PYANSYS_PYPI_PRIVATE_PAT>` on line 319 is usually copied pasted by the user, it also triggers an error bc of the <> characters

**Review by kellymore — COMMENTED** [review]
` Otherwise ``twine``
attempts to upload the package to the public PyPI repository.` on line 262, idk if this is true or not. This may have also triggered bc I was being a trouble maker and testing to see if it'll work on public

**Comment by RobPasMue** [comment]
> because `set TWINE_PASSWORD=<PYANSYS_PYPI_PRIVATE_PAT>` on line 319 is usually copied pasted by the user, it also triggers an error bc of the <> characters

Well... we have to somehow show that ``<PYANSYS_PYPI_PRIVATE_PAT>`` has to be substituted. One approach we follow in other repos is:

```rst
    If access to this package registry is needed, email `pyansys.support@ansys.com <mailto:pyansys.support@ansys.com>`_
    to request access. The PyAnsys team can provide you a read-only token to be inserted in ``${PRIVATE_PYPI_ACCESS_TOKEN}``.
    Once you have it, run the following command:

    .. code:: bash

        pip install my-package --index-url=https://${PRIVATE_PYPI_ACCESS_TOKEN}@pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/simple/

```

That way we clearly state that it has to be completely subsitituted. But we should not be posting the value of the token in any case so... the solution ain't straightforward 😄 you're always going to get an error if you try to directly copy-paste. The only thing we can do is clarify even more what precise combination of characters and words has to be substituted from the command.

**Pull request merged**